draft-ietf-lamps-cms-kyber.md

```diff
@@ -93,11 +101,11 @@ informative:
       org: National Institute of Standards and Technology
     title: "Cryptographic Module Validation Program"
     date: 2016
-  RFC5911:
 
 --- abstract
 
-Module-Lattice-Based Key-Encapsulation Mechanism (ML-KEM) is a quantum-resistant key-encapsulation mechanism (KEM). Three parameter sets for the ML-KEM algorithm are specified by NIST in FIPS 203. In order of increasing security strength (and decreasing performance), these parameter sets are ML-KEM-512, ML-KEM-768, and ML-KEM-1024. This document specifies the conventions for using ML-KEM with the Cryptographic Message Syntax (CMS) using the KEMRecipientInfo structure.
+Module-Lattice-Based Key-Encapsulation Mechanism (ML-KEM) is a quantum-resistant key-encapsulation mechanism (KEM). Three parameter sets for the ML-KEM algorithm are specified by the US National Institute of Standards and Technology (NIST) in FIPS 203. In order of increasing security strength (and decreasing performance), these parameter sets are ML-KEM-512, ML-KEM-768, and ML-KEM-1024. This document specifies the conventions for using ML-KEM with the Cryptographic Message Syntax (CMS) using the KEMRecipientInfo structure defined in “Using Key Encapsulation
+Mechanism (KEM) Algorithms in the Cryptographic Message Syntax (CMS)” {{!RFC9629}}.
```

Comment:
```suggestion
Mechanism (KEM) Algorithms in the Cryptographic Message Syntax (CMS)” (RFC 9629).
```

Per this part from RFC7322:

>    Similarly, the Abstract should be complete in itself.  It will appear
>    in isolation in publication announcements and in the online index of
>    RFCs.  Therefore, the Abstract must not contain citations.